BeagleBone_black/src/motor_control/main.hpp

```diff
@@ -38,24 +39,24 @@ class Main: public Thread {
  public:
   explicit Main(uint8_t id, Logger& log);
   void run() override;
+
+ private:
   void setupMotors();
   void accelerateMotors();
   void decelerateMotors();
   void stopMotors();
-  int32_t calculateAccelerationRPM(uint32_t velocity);
-  int32_t calculateDecelerationRPM(uint32_t velocity);
-
- private:
-  Motor* motor;
-  int32_t rpm;
-  data::Data& data = data::Data::getInstance();
-  data::Motors motor_data;
-  data::StateMachine state;
-  data::Navigation nav;
-  bool motorsSetUp;
-  bool motorFailure;
+  int32_t calculateAccelerationRPM(NavigationType velocity);
+  int32_t calculateDecelerationRPM(NavigationType velocity);
+  data::Data& data_;
+  Motor motor_;
+  data::Motors motor_data_;
+  data::StateMachine state_;
+  data::Navigation nav_;
+  int32_t rpm_;
+  bool motorsSetUp_;
+  bool motorFailure_;
```

Comment:
No uppercase letters in variable names! https://hyp-ed.github.io/styleguide/#Variable_Names